pywbem/cim_obj.py

```diff
@@ -3723,28 +3781,30 @@ def __init__(self, name, type, value=None, is_array=False,
             attribute.
 
           overridable (:class:`py:bool`):
-            If not `None`, specifies whether the qualifier value is overridable
-            in subclasses.
+            If not `None`, defines the flavor that defines whether the
+            qualifier value is overridable in subclasses.
```

Comment:
For all flavor parameters, see my comments above, on the corresponding attributes.